build.gradle

```diff
@@ -9,7 +9,6 @@ plugins {
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
-ext['logbackVersion'] = '1.1.11'
```

Comment:
Out of curiosity. Why was this removed?

Reply:
unused variable, cause confusion with the real version of logback

Reply:
Actually, `Logback` comes along with the `logback-access-spring-boot-starter` Spring Boot dependency. This line is changing the version of Logback that Spring Boot adds. It's better to use the same version that comes with Spring Boot unless there's a security vulnerability that needs to be solved. This change should be fine if that's the case 🙂